media/au.ksy

```diff
@@ -0,0 +1,61 @@
+meta:
+  id: au
+  title: AU audio format
+  file-extension:
+    - au
+    - snd
+  xref:
+    justsolve:
+      - AU
+    mime:
+      - audio/basic
+    pronom:
+      - x-fmt/139
+    wikidata:
+      - Q672985
+  license: CC0-1.0
+  ks-version: 0.9
+  endian: be
+doc: |
+  Test files for AU can be found in any Python release in the
+  directory Lib/test/sndhdrdata/
```

Comment:
1. a direct link should be put
2. Gonna be removed: https://www.python.org/dev/peps/pep-0594/#sunau . Tests should likely be moved to https://github.com/tiran/legacylib/tree/master/sunau tool but it is not done yet